### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by yanaost**
https://yanaost.github.io/layout_calendar/
https://yanaost.github.io/layout_calendar/report/html_report/

### CODE REVIEW

File: src/styles/blocks/calendar.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      .calendar__cell:nth-child(#{$i}) {
+        background-color: $day-color;
+        border: 1px solid black;
+        &:hover {
+          background-color: pink;
+        }
+        &::before {
+          content: '#{$i}';
+          font-family: $number-font;
+          size: $number-size;
```
Comment (AmbeMO) [inline src/styles/blocks/calendar.scss]: font-size

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  "sat" $margin-left-var * 5,
+  "sun" $margin-left-var * 6;
+
+$month-days:
+  "28" 28,
+  "29" 29,
+  "30" 30,
+  "31" 31;
+
+.calendar {
```
Comment (AmbeMO) [inline src/styles/blocks/calendar.scss:1]: calendar should be in vertical center too

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  padding: 10px;
+  margin: 0 auto;
+
+  &__cell {
+    box-sizing: border-box;
+    width: $day-size;
+    height: $day-size;
+
+    &:hover {
+      cursor: pointer;
```
Comment (AmbeMO) [inline src/styles/blocks/calendar.scss:20]: its optional but u can also add translateY or similar
and add transition

File: src/styles/blocks/calendar.scss [hunk 4]
```diff
@@ -0,0 +1,63 @@
+$weekday:
```
Comment (AmbeMO) [inline src/styles/blocks/calendar.scss]: it can be in another file, but maybe optional. Its about variables

**Review by AmbeMO — CHANGES_REQUESTED** [review]
small fix

File: src/styles/blocks/calendar.scss (hunk 4 above)
Comment (AlinaMatuschak) [reply src/styles/blocks/calendar.scss]: Agree with @AmbeMO , create file for variables

File: src/styles/blocks/calendar.scss [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  "29" 29,
+  "30" 30,
+  "31" 31;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+  gap: 1px;
+  width: $day-size * 7 + 20px;
```
Comment (AlinaMatuschak) [inline src/styles/blocks/calendar.scss]: 7 is a magic number. Create variable for this num

**Review by AlinaMatuschak — APPROVED** [review]
Also, believe that you can move cell up by 20px on hover

**Pull request closed without merging**